ambry-clustermap/src/main/java/com/github/ambry/clustermap/ClusterMapUtils.java

```diff
@@ -225,25 +228,52 @@ public static ReplicaSealStatus convertPartitionStateToReplicaSealSatus(Partitio
   }
 
   /**
-   * Choose a random {@link PartitionId} for putting the metadata chunk and return it.
-   * @param partitionClass the partition class to choose partitions from.
```

Comment:
Did you accidentally remove the java doc of "partitionClass" and "partitionIdsToExclude"?